closure/goog/i18n/numberformat.js

```diff
@@ -589,8 +589,8 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
  *     This function will add its formatted pieces to the array.
  * @private
  */
-goog.i18n.NumberFormat.prototype.subformatFixed_ =
-    function(number, minIntDigits, parts) {
+goog.i18n.NumberFormat.prototype.subformatFixed_ = function(
```

Comment:
In general, I'm not too familiar with the i18n libs. Can you explain to me how this format method changes its behavior based on locale? I.e., how does it decide to format in the Indian comma placement vs. not?